romancal/outlier_detection/tests/test_outlier_detection.py

```diff
@@ -140,6 +151,12 @@ def test_outlier_init_default_parameters(pars, base_image):
     assert step.resample_suffix == f"_outlier_{pars['resample_suffix']}.asdf"
 
 
+# FIXME: This test checks if the median image exists on disk after outlier detection.
+# Howver "save_intermediate_results=False" so this file should not be saved even if
+# in_memory=False (which only means the file will temporarily be produced if needed).
+@pytest.mark.skip(
```

Comment:
I opened an issue to track this skip:

https://github.com/spacetelescope/romancal/issues/1303

It's possible that setting `save_intermediate_files` to True would be enough to fix this but I did not do so in this PR to try to limit the scope. If it's preferred to make the fix vs add this skip I'm happy to do so.